proto-docs/object.md

```diff
@@ -1044,6 +1047,8 @@ prefix to the name. Here is the list of fields available via this prefix:
   16 byte UUIDv4 used to identify the split object hierarchy parts
 * $Object:split.first \
   object_id of the first part in split chain; non-acceptable for deprecated V1 split scheme
+* $Object:target \
```

Comment:
with `__NEOFS__TARGET` attribute this wouldnt've been needed. Any reason to not do it?

Reply:
this is a core feature for me with a certain type and non-string original representation. i feel it the same way as oid or other fields. likely all the necessary fields can be placed in the attributes in their string form, and SN may be forced to validate them, but what for?

Reply:
the same way, locking many objects with a single LOCK object will not be acceptable. non-compatible either

Reply:
@roman-khimov, thoughts?